integration-tests/src/test/java/org/zowe/apiml/gatewayservice/VersionEndpointTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.gatewayservice;
+
+import io.restassured.RestAssured;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.zowe.apiml.util.config.ConfigReader;
+import org.zowe.apiml.util.config.GatewayServiceConfiguration;
+
+import static io.restassured.RestAssured.given;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.Matchers.is;
+
+public class VersionEndpointTest {
+
+    private String requestString;
+
+    @BeforeClass
+    public static void beforeClass() {
+        RestAssured.useRelaxedHTTPSValidation();
+    }
+
+    @Before
+    public void setUp() {
+        GatewayServiceConfiguration serviceConfiguration = ConfigReader.environmentConfiguration().getGatewayServiceConfiguration();
+        String scheme = serviceConfiguration.getScheme();
+        String host = serviceConfiguration.getHost();
+        int port = serviceConfiguration.getPort();
+        requestString = String.format("%s://%s:%s%s", scheme, host, port, "/version");
```

Comment:
Integration test is failing. `requestString` should be updated to contain also either `/gateway` or `/api/v1/gateway` I guess.

Reply:
Good point! Thanks for catched it!